pico/api/src/main/java/io/helidon/pico/api/ExternalContracts.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.pico.api;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Placed on the implementation of a service as an alternative to using a {@link Contract}.
+ *
+ * Use this annotation when it is impossible to place an annotation on the interface itself - for instance of the interface comes
+ * from a 3rd party library/provider.
+ */
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
We should not need runtime retention, as we plan to do everything at compile time (this is valid for all the annotations).

Reply:
I've already advised the developer to not include this module into compile scope; my thinking is to leave it here for anyone that still wants to use reflection, etc.

Reply:
I think we should not do that - it will make the class file bigger and we want to process it compile time always.